wiki/Play_Styles/Drag/en.md

```diff
@@ -0,0 +1,18 @@
+---
+tags:
+  - tablet
+  - dragging
+  - play style
+---
+
+<!-- This article is a stub -->
+
+# Drag
+
+**Drag** is a term used to describe the act of navigating (or "aiming") a [tablet](/wiki/Glossary#tablet) pen by physically touching the pen to the surface of the tablet and "dragging" it to the desired location (as opposed to [hovering](/wiki/Play_Styles/Hover)). This style is used by players of all skill levels and is easy to learn.
+
+*Note: Though players may argue, at the end of the day, it's all down to personal preference; there's no one play style/input method that's objectively better than the other.*
```

Comment:
Do we really want this note here? It should already be implied that this is the case.

Reply:
will remove it soon™